internal/home/upgrade_test.go

```diff
@@ -558,6 +560,18 @@ func TestUpgradeSchema12to13(t *testing.T) {
 	}
 }
 
+func TestUpgradeSchema12to13Regression(t *testing.T) {
```

Comment:
I feel like a better regression test would try to migrate a configuration file going all the way back to version `1`.  Please add a TODO [assigned to me][1] about that.

[1]: https://github.com/AdguardTeam/CodeGuidelines/blob/master/Go/Text.md#li-2249ae11